README.md

```diff
@@ -1,35 +1,43 @@
----
-page_type: sample
-languages:
-- azurecli
-- python
-products:
-- azure-machine-learning
-description: Top-level directory for official Azure Machine Learning sample code and examples.
----
-
 # Azure Machine Learning examples
 
-[![smoke](https://github.com/Azure/azureml-examples/workflows/smoke/badge.svg)](https://github.com/Azure/azureml-examples/actions/workflows/smoke.yml)
-[![Python code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![license: MIT](https://img.shields.io/badge/License-MIT-purple.svg)](LICENSE)
 
 Welcome to the Azure Machine Learning examples repository!
 
-## Contents
-
-|directory||description|
-|-|-|-|
-|[`.github`](.github)||GitHub files like issue templates and actions workflows.|
-|[`cli`](cli)||Azure Machine Learning CLI v2 examples.||
-|[`sdk`](sdk)||Azure Machine Learning SDK v2 examples.||
-||[`python`](sdk/python/)| Azure Machine Learning Python SDK v2 examples.|
-||[`dotnet`](sdk/dotnet)| Azure Machine Learning .Net SDK v2 examples.|
-|[`setup`](setup)||Folder with setup scripts|
-||[`setup-ci`](setup/setup-ci)|Setup scripts to customize and configure ||an Azure Machine Learning compute instance.|
-||[`setupdsvm`](setup/setup-dsvm/RStudio/)|Setup RStudio on Data Science Virtual Machine (DSVM)|
-||[`setup-repo`](setup/setup-repo)|Setup scripts for Azure/azureml-examples.|
-|[`tutorials`](tutorials/)||Azure Machine Learning end to end Python SDK v2 tutorials|
+## About This Repository
+
+The azureml-examples repository contains examples and tutorials to help you learn how to use Azure Machine Learning
+(Azure ML) services and features.
+
+
+### Getting Started
+
+If you're getting started with Azure ML, consider working through our [tutorials] for the v2 Python SDK. You may
+also want to read through our [documentation](#supplementary-documentation).
+
+### SDKs
+
+The `sdk/` folder houses the examples for the Azure ML SDKs across several languages.
+
+We have an extensive collection of examples for the [Azure ML Python SDK v2][azure cli ml extension v2 overview] in
+[`sdk/python`][azureml python sdk v2 examples].
+
+We also offer some examples for our SDKs in other languages:
+
+* .NET: [`sdk/dotnet`][azureml dotnet sdk v2 examples]
```

Comment:
Since we're calling out languages - we should do the same for R somewhere in this file. Here's the location of our R folder for CLI: https://github.com/Azure/azureml-examples/tree/main/cli/jobs/single-step/r

Reply:
What kind of callout are you looking for specifically? Something like "For users interested in submitting Azure ML jobs that run R, see [these examples](https://github.com/Azure/azureml-examples/tree/main/cli/jobs/single-step/r)"

Reply:
Resolved by https://github.com/Azure/azureml-examples/pull/2697/commits/c487e3aa5329c13e0501629c912aff9bd2448574